### ISSUE TITLE
Module not found: Can't resolve 'react' in '/home/fqye/node_modules/react-redux/es/components'

### ISSUE THREAD

**Issue by broye** [zeit/next.js#8597]
I am trying out nextjs with Redux using next-redux-wrapper. 

I encountered this strange error below.

I am running a standard next.js 9.0.5 and TS, with a simple next.config.js for sass.

**Error:**

```
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
[ error ] /home/fqye/node_modules/react-redux/es/components/Provider.js
Module not found: Can't resolve 'react' in '/home/fqye/node_modules/react-redux/es/components'
[ wait ]  compiling ...
[ error ] /home/fqye/node_modules/react-redux/es/components/Provider.js
Module not found: Can't resolve 'react' in '/home/fqye/node_modules/react-redux/es/components'
[ event ] build page: /next/dist/pages/_error
[ wait ]  compiling ...
[ error ] /home/fqye/node_modules/react-redux/es/components/Provider.js
Module not found: Can't resolve 'react' in '/home/fqye/node_modules/react-redux/es/components'
Error: Cannot find module '/home/fqye/projects/oqa/web/.next/build-manifest.json'
Require stack:
- /home/fqye/projects/oqa/web/node_modules/next-server/dist/server/load-components.js
- /home/fqye/projects/oqa/web/node_modules/next-server/dist/server/api-utils.js
- /home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js
- /home/fqye/projects/oqa/web/node_modules/next/dist/server/next-dev-server.js
- /home/fqye/projects/oqa/web/node_modules/next/dist/server/next.js
- /home/fqye/projects/oqa/web/node_modules/next/dist/server/lib/start-server.js
- /home/fqye/projects/oqa/web/node_modules/next/dist/cli/next-dev.js
- /home/fqye/projects/oqa/web/node_modules/next/dist/bin/next
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.loadComponents (/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/load-components.js:21:9)
    at DevServer.findPageComponents (/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js:308:40)
    at DevServer.renderErrorToHTML (/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js:362:35)
    at DevServer.renderErrorToHTML (/home/fqye/projects/oqa/web/node_modules/next/dist/server/next-dev-server.js:16:466)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async DevServer.render (/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js:284:22)
    at async Object.fn (/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js:173:21)
    at async DevServer.run (/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js:259:17) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/load-components.js',
    '/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/api-utils.js',
    '/home/fqye/projects/oqa/web/node_modules/next-server/dist/server/next-server.js',
    '/home/fqye/projects/oqa/web/node_modules/next/dist/server/next-dev-server.js',
    '/home/fqye/projects/oqa/web/node_modules/next/dist/server/next.js',
    '/home/fqye/projects/oqa/web/node_modules/next/dist/server/lib/start-server.js',
    '/home/fqye/projects/oqa/web/node_modules/next/dist/cli/next-dev.js',
    '/home/fqye/projects/oqa/web/node_modules/next/dist/bin/next'
  ]
}
```

**next.config.js**

```
const withSass = require('@zeit/next-sass')
module.exports = withSass({
  /* config options here */
})
```

**Comment by timneutkens** [comment]
Please follow the issue template. A full reproduction is needed for us to look into your issue.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
